content/en/learn/governance/module-psm.md

```diff
@@ -17,29 +17,29 @@ Scope: Each PSM interacts with a single underlying vault type.
 
 A Peg Stability Module allows users to swap a given collateral type directly for DAI at a fixed rate, rather than borrowing DAI. The PSM contract was designed with stablecoin collateral in mind, allowing users to swap other stablecoins for DAI at a fixed rate to aid with keeping DAI pegged to 1 dollar.
 
-A PSM operates similarly to a regular vault type with a zero stability fee and a liquidation ratio of 100% that can only be accessed through a user-facing smart contract containing the relevant swap functions. Unlike a regular vault, a user of a PSM does not retain ownership of the asset and borrow DAI, instead they swap the asset directly for DAI.
+A PSM operates similarly to a regular vault type with a zero stability fee and a liquidation ratio of 100% that can only be accessed through a user-facing smart contract containing the relevant swap functions. Unlike a regular vault, PSM's user doesn't retain ownership of the asset and borrow DAI, instead they swap the asset directly for DAI.
```

Comment:
"Unlike the case with regular vaults, users of PSM don't retain..."?

Reply:
changed, thanks!